Rules/UseConsistentWhitespace.cs

```diff
@@ -364,6 +375,47 @@ private IEnumerable<DiagnosticRecord> FindOpenParenViolations(TokenOperations to
             }
         }
 
+        private IEnumerable<DiagnosticRecord> FindParameterViolations(Ast ast)
+        {
+            var commandAsts = ast.FindAll(
+                    testAst => testAst is CommandAst, true);
+            foreach (CommandAst commandAst in commandAsts)
+            {
+                var commandParameterAstElements = commandAst.FindAll(testAst => true, searchNestedScriptBlocks: false).ToList();
+                for (int i = 0; i < commandParameterAstElements.Count - 1; i++)
+                {
+                    var leftExtent = commandParameterAstElements[i].Extent;
+                    var rightExtent = commandParameterAstElements[i + 1].Extent;
+                    var extentsAreOnSameLine = leftExtent.EndLineNumber == rightExtent.StartLineNumber;
+                    if (!extentsAreOnSameLine)
+                    {
+                        continue;
+                    }
+                    var expectedStartColumnNumberOfRightExtent = leftExtent.EndColumnNumber + 1;
```

Comment:
```suggestion
```

Reply:
I applied suggestions at first, which broke the build as the variable is used a 2nd time as well. By looking at it again, I reverted the suggestion because having a variable for it makes the code more readable in this case, especially when some maths is needed a few lines below.